jupyterlab_server/config.py

```diff
@@ -370,6 +370,8 @@ def _get_config_manager(level: str, include_higher_levels: bool = False) -> Conf
     """Get the location of config files for the current context
     Returns the string to the environment
     """
+    from jupyter_core.paths import ENV_CONFIG_PATH
```

Comment:
Restoring the comment from previous version too:

```suggestion
    # Delayed import since this gets monkey-patched in tests
    from jupyter_core.paths import ENV_CONFIG_PATH
```